tests/beaker/tests/create_lv_with_param_alloc.rb

```diff
@@ -0,0 +1,82 @@
+require 'master_manipulator'
+require 'lvm_helper'
+require 'securerandom'
+
+test_name "FM-4579 - C96574 - create logical volume without parameter 'alloc'"
+
+#initilize
+pv = '/dev/sdc'
+vg = ("VolumeGroup_" + SecureRandom.hex(2))
+lv = [("LogicalVolume_" + SecureRandom.hex(3)), ("LogicalVolume_" + SecureRandom.hex(3)), \
```

Comment:
We discussed renaming these variables from the abbreviations to more descriptive names. Did you change your mind?